README.md

```diff
@@ -245,10 +245,104 @@ If you experienced a compatibility issue with any of the following plugins, plea
 - [mini-css-extract-plugin](https://github.com/webpack-contrib/mini-css-extract-plugin)
 - [HtmlWebpackPlugin](https://github.com/jantimon/html-webpack-plugin)
 
+### options.contentScript.experimental_output
+
+**This is an experimental API. API might change at any time. Please provide feedback!**
+
+#### TLDR: How to use this
+
+##### string
+
+If you don't strictly rely on [run_at](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/content_scripts#run_at),
```

Comment:
what would happen for cases where users have `run_at` defined?

Reply:
`cs.js` is like this:

```js
;(async () => {
	const getURL = typeof browser === "object" ? browser.runtime.getURL : chrome.runtime.getURL;
	await import(getURL("runtime.js"));
	await import(getURL("test_txt-util_js.js"));
	await import(getURL("content.js"));
})();
null;
```

therefore it _will not_ run at the very beginning (which may be a hard limit for some content scripts)